[2024-01-13T21:39:39.107+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.bigquery_external_table_task manual__2024-01-13T21:23:58.788584+00:00 [queued]>
[2024-01-13T21:39:39.115+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.bigquery_external_table_task manual__2024-01-13T21:23:58.788584+00:00 [queued]>
[2024-01-13T21:39:39.116+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 2
[2024-01-13T21:39:39.128+0000] {taskinstance.py:2192} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2024-01-13 21:23:58.788584+00:00
[2024-01-13T21:39:39.136+0000] {standard_task_runner.py:60} INFO - Started process 1066 to run task
[2024-01-13T21:39:39.139+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'bigquery_external_table_task', 'manual__2024-01-13T21:23:58.788584+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpr3pkt27o']
[2024-01-13T21:39:39.143+0000] {standard_task_runner.py:88} INFO - Job 15: Subtask bigquery_external_table_task
[2024-01-13T21:39:39.156+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-01-13T21:39:39.185+0000] {task_command.py:423} INFO - Running <TaskInstance: data_ingestion_gcs_dag.bigquery_external_table_task manual__2024-01-13T21:23:58.788584+00:00 [running]> on host 6bca031b1185
[2024-01-13T21:39:39.236+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_gcs_dag' AIRFLOW_CTX_TASK_ID='bigquery_external_table_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-13T21:23:58.788584+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-13T21:23:58.788584+00:00'
[2024-01-13T21:39:39.240+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-01-13T21:39:39.241+0000] {connection.py:234} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-01-13T21:39:39.243+0000] {base.py:83} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-01-13T21:39:40.061+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=bigquery_external_table_task, execution_date=20240113T212358, start_date=20240113T213939, end_date=20240113T213940
[2024-01-13T21:39:40.100+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-13T21:39:40.116+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
